workspaces/marketplace/plugins/marketplace/package.json

```diff
@@ -43,6 +43,7 @@
     "@monaco-editor/react": "^4.7.0",
     "@mui/icons-material": "^5.16.7",
     "@mui/material": "^5.12.2",
+    "@mui/styles": "5.16.14",
```

Comment:
where is this new dependency used? also ideally we should be using `sx` instead as `@mui/styles` [is deprecated](https://www.npmjs.com/package/@mui/styles)